recipes-bsp/u-boot/u-boot_%.bbappend

```diff
@@ -31,4 +46,30 @@ do_deploy:append:freedom-u540() {
     fi
 }
 
+do_configure[depends] += "opensbi:do_deploy"
+
+do_compile:prepend:ae350-ax45mp() {
+    export OPENSBI=${DEPLOY_DIR_IMAGE}/fw_dynamic.bin
+}
+
+do_configure:prepend:ae350-ax45mp() {
+    if [ -f "${WORKDIR}/tftp-mmc-boot.txt" ]; then
+        sed -i -e 's,@SERVERIP@,${TFTP_SERVER_IP},g' ${WORKDIR}/tftp-mmc-boot.txt
+        mkimage -A riscv -O linux -T script -C none -n "U-Boot boot script" \
+            -d ${WORKDIR}/tftp-mmc-boot.txt ${WORKDIR}/boot.scr.uimg
+    fi
+}
```

Comment:
Are you using TFTP boot?

Reply:
Yes we use u-boot to load kernel image from TFTP server.